@xen-orchestra/fs/src/s3.js

```diff
@@ -13,12 +15,14 @@ const MAX_PART_SIZE = 1024 * 1024 * 1024 * 5 // 5GB
 const MAX_PARTS_COUNT = 10000
 const MAX_OBJECT_SIZE = 1024 * 1024 * 1024 * 1024 * 5 // 5TB
 const IDEAL_FRAGMENT_SIZE = Math.ceil(MAX_OBJECT_SIZE / MAX_PARTS_COUNT) // the smallest fragment size that still allows a 5TB upload in 10000 fragments, about 524MB
+
+const USE_SSL = true
```

Comment:
What's this for?

Reply:
quick test against non-SSL server

Reply:
it will come through the remote URL soon, it's a first step.

another preview is here: https://github.com/vatesfr/xen-orchestra/pull/5579/files